BoolParamterTest/test_ini.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import os
+from pathlib import Path
+
+# Dynamically add the src folder to sys.path so Python can find it
+sys.path.append(str(Path(__file__).resolve().parent.parent / "src"))
+
+# Now import ParameterManager from the correct module
+from workflow.ParameterManager import ParameterManager  
+
+# Create ParameterManager instance
+pm = ParameterManager(workflow_dir=Path("./workflow"))
+
+# Path to a sample INI file (Modify if needed)
+ini_file_path = "workflow/test.ini"
+
+# Check if the INI file exists before loading
+if not Path(ini_file_path).exists():
+    print(f"⚠️ Warning: The INI file '{ini_file_path}' does not exist. Creating a sample one...")
+    # Create a sample INI file with boolean and string parameters
+    ini_content = """<?xml version="1.0" encoding="UTF-8"?>
+<PARAMETERS version="2.0">
+    <ITEM name="enable_feature" value="true" type="bool"/>
+    <ITEM name="debug_mode" value="false" type="bool"/>
+    <ITEM name="execution_mode" value="strict" type="string"/>
```

Comment:
I think it would make more sense to set the value to `true` or `false`. String values are often `true` or `false` but this should not result in them being casted to boolean.